docs/faq.md

```diff
@@ -192,3 +192,18 @@ In the EarlGrey test target, import the application's app delegate header. In `s
     [delegate resetApplicationForTesting];
 }
 ```
+
+**How do I increase the speed of `grey_allOf`?**
```

Comment:
Shouldn't the question be more along the lines of what you initially reported:
Takes too long to do XXX or screen appears frozen for X seconds?